tests/phpunit/api/v3/MailingGroupTest.php

```diff
@@ -16,94 +16,65 @@
  * @group headless
  */
 class api_v3_MailingGroupTest extends CiviUnitTestCase {
-  protected $_groupID;
-  protected $_email;
-  protected $_apiversion;
 
   public function setUp(): void {
     parent::setUp();
-    $this->useTransaction(TRUE);
-    $this->_apiversion = 3;
-    $this->_groupID = $this->groupCreate();
-    $this->_email = 'test@test.test';
-  }
-
-  /**
-   * Test civicrm_mailing_group_event_subscribe with wrong params.
-   */
-  public function testMailerGroupSubscribeWrongParams() {
-    $params = [
-      'email' => $this->_email,
-      'group_id' => 'Wrong Group ID',
```

Comment:
Which one is this a duplicate of? I don't see any others testing the wrong group id?

Reply:
@demeritcowboy generally we test values not being valid in SyntaxConformanceTest so when we are just testing the api wrapper I tend to remove them (especially failure tests)